src/Serilog.Sinks.AzureTableStorage/project.json

```diff
@@ -11,17 +11,32 @@
     },
     "dependencies": {
         "Serilog": "2.0.0",
-        "Serilog.Sinks.PeriodicBatching": "2.0.0",
-        "WindowsAzure.Storage": "4.3.0"
+        "Serilog.Sinks.PeriodicBatching": "2.0.0"
     },
     "buildOptions": {
         "keyFile": "../../assets/Serilog.snk"
     },
     "frameworks": {
-        "net4.5": {
-            "frameworkAssemblies": {
-                "System.Configuration": ""
-            }
+      "net4.5": {
+        "dependencies": {
+          "WindowsAzure.Storage": "4.3.0"
+        },
+        "frameworkAssemblies": {
+          "System.Configuration": ""
         }
+      },
+      "netstandard1.3": {
+        "buildOptions": {
+          "define": [
+            "WINDOWS_AZURE_STORAGE_ASYNC_ONLY_APIS"
```

Comment:
I rekon we might as well update to 7.2.0 and async for the whole library for full framework and non-full-framework. There isn't any particular reason why they weren't async to start with tbh. I don't think it's necessarily even a breaking change, but can't hurt to bump major version.